### ISSUE TITLE
Issues with input_number " TypeError: null is not an object" on Safari

### ISSUE THREAD

**Issue by VirtualL** [home-assistant/ui-schema#151]
Hi, I getting this Warning : 
```https://MYSERVER.COM/frontend_latest/eb618cf11a4eca6918ea.chunk.js:266:14994 TypeError: null is not an object (evaluating 'this.shadowRoot.querySelector("paper-slider").offsetWidth')```

Its a only apearing in Safari (I have the latest Version 11.1.1 (13605.2.8)) , the same config in google Chrome have no warnings . 

The config:
```
      - type: entities
        title: "Boiler Timer"
        show_header_toggle: false
        entities:
          - switch._mqtt_pow1
          - type: divider
          - script.timer_off_boiler
          - input_number.slider_boiler
          - sensor.pow1_current
```

If I will remove the input_number entities , the warning will disappear 

Thanks
